.circleci/config.yml

```diff
@@ -20,17 +20,9 @@ jobs:
           command: |
             docker-compose build
             docker-compose up -d
-      - run:
-          name: install dockerize
-          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
-          environment:
-            DOCKERIZE_VERSION: v0.3.0
-      - run:
-          name: Wait for db
-          command: dockerize -wait http://localhost:5000/dataset -timeout 120s
-      - run:
-          name: Run ckanext-datajson Tests
-          command: |
+            sleep 40
```

Comment:
The whole point of using dockerize was to avoid `sleep` like this which can be unreliable. Why not handle this properly?

Reply:
@adborden I just [re-add](https://github.com/GSA/ckanext-datagovtheme/pull/62/commits/78b0f5936b4150966225646a927efecf8768b00d) dockerize. 
We now just need to publish the `catalog-next` docker image to test this